starlette_admin/translations/tr/LC_MESSAGES/admin.po

```diff
@@ -157,10 +157,18 @@ msgstr "Hesabınıza giriş yapın"
 msgid "Username"
 msgstr "Kullanıcı adı"
 
+#: starlette_admin/templates/login.html:36
+msgid "Enter username"
+msgstr "Kullanıcı adı"
+
 #: starlette_admin/templates/login.html:43
 msgid "Password"
 msgstr "Şifre"
 
+#: starlette_admin/templates/login.html:47
+msgid "Enter password"
+msgstr "Şifre"
```

Comment:
Can you confirm if this is accurate?

Reply:
Nope, I'll fix it.

Reply:
Yep, I checked some major Turkish websites. They are not saying "enter username" or "put username", just "username". For context, we should follow their approach.

Reply:
Yes, it's better.